enterprise/frontend/src/metabase-enterprise/whitelabel/components/ColorSchemeWidget.jsx

```diff
@@ -83,7 +87,7 @@ const ColorSchemeWidget = ({ setting, onChange }) => {
                 </td>
                 <td>
                   <span className="mx2 text-grey-4">
-                    {properties.description}
+                    {jt`${properties.description}`}
```

Comment:
This translates the highlighted text.
![image](https://user-images.githubusercontent.com/1937582/162979447-a83ea11f-0a22-4b47-b184-78e665f4a51b.png)